### ISSUE TITLE
New character models needed for MT after #6007

### ISSUE THREAD

**Issue by lhofhansl** [Wuzzy2/MineClone2-Bugs#337]
After [#6007](https://github.com/minetest/minetest/pull/6007) new character models are needed.
Happy to create them.

**Comment by lhofhansl** [comment]
Actually it's just the 3d-armor models that need be changed. See [3d-armor #112](https://github.com/stujones11/minetest-3d_armor/pull/112)

**Comment by Wuzzy2** [comment]
This is correct. I wait for 0.5.0 to be released, then fixing this issue has top priority. But not now, I do not want to run after moving target.
